### PR TITLE
[DCFS-11] Sign-Up Form Validation

### PR DESCRIPTION
**Pull request by kemysola**
## Task: `Dom Manipulation`
- `Changes you made to given task`
- @adaugochi Good afternoon ma, Kindly review.Thank you.

### CODE REVIEW

**Review by adaugochi — CHANGES_REQUESTED** [review]
` firstNameValidation,  secondNameValidation emailValidation and mobilelValidation` functions are all doing the same thing. It more like you are repeating yourself over and over again. The only difference here is the `name field, error message and the regex expression`. Why not create a single function that does this and pass what makes them different as a parameter

**Comment by kemysola** [comment]
> ` firstNameValidation, secondNameValidation emailValidation and mobilelValidation` functions are all doing the same thing. It more like you are repeating yourself over and over again. The only difference here is the `name field, error message and the regex expression`. Why not create a single function that does this and pass what makes them different as a parameter

Good Morning ma @adaugochi .Thank you i have corrected it .Kindly review. Thank you.

File: submissions/adekemiAwomolo/dom/signup.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+            event
+        );
+        required(
+            confirm_password,
+            passwordReg,
+            "Please Confirm Your Password",
+            'Password do not match',
+            event
+        );
+        passwordValidation(password, confirm_password, event);
```
Comment (adaugochi) [inline submissions/adekemiAwomolo/dom/signup.js:74]: I can't find this `passwordValidation` function

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Review by adaugochi — CHANGES_REQUESTED** [review]
You are also meant to check if `password` and `confirmPAssword` match

**Comment by kemysola** [comment]
> You are also meant to check if `password` and `confirmPAssword` match

@adaugochi Thank you ma, i have done the check. Kindly approve ma.

File: submissions/adekemiAwomolo/dom/signup.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+            event.preventDefault();
+            return false;
+        }
+        else{
+            field.nextElementSibling.innerHTML="";
+            return true;
+        }
+    
+}
+function emailValidation(event){
```
Comment (adaugochi) [inline submissions/adekemiAwomolo/dom/signup.js:31]: This function name is misleading. This function is not validating email

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/adekemiAwomolo/dom/signup.js (hunk 2 above)
Comment (kemysola) [reply submissions/adekemiAwomolo/dom/signup.js]: Good morning @adaugochi  i have corrected the function name, kindly review.Thank you

**Review by adaugochi — APPROVED** [review]

**Pull request merged**